app/javascript/onboarding/components/EmailListTermsConditionsForm.jsx

```diff
@@ -13,10 +13,10 @@ class EmailTermsConditionsForm extends Component {
     this.checkRequirements = this.checkRequirements.bind(this);
 
     this.state = {
-      checked_code_of_conduct: false,
-      checked_terms_and_conditions: false,
-      email_membership_newsletter: true,
-      email_digest_periodic: true,
+      checkedCodeOfConduct: false,
+      checkedTermsAndConditions: false,
+      emailMembershipNewsletter: true,
+      emailDigestPeriodic: true,
```

Comment:
I think we should avoid making these snake_case to camelCase changes in this PR and tackle it in another PR based on @maestromac's feedback. Would you mind undoing this change? I'd love to merge in the other linting changes.

Reply:
No problem at all, thanks for reviewing @jacobherrington! Resubmitting now.